lib/membrane_live_web/plugs/bearer_plug.ex

```diff
@@ -0,0 +1,26 @@
+defmodule MembraneLiveWeb.Plugs.BearerPlug do
+  @moduledoc """
+  Plug responsible for fetching jwt token from HTTP request,
+  decoding it and assigning decoded user_id to conn object
+  """
+  import Plug.Conn
+  alias MembraneLive.Tokens
+  alias MembraneLiveWeb.FallbackController
+
+  def init(default), do: default
+
+  def call(conn, _default) do
+    case Enum.find(conn.req_headers, fn elem -> match?({"bearer", _}, elem) end) do
+      {"bearer", jwt} ->
+        %{"user_id" => user_id} = Tokens.custom_decode(jwt)
```

Comment:
- the header key we are looking for is `authorization`
 - the header vallue is always "Bearer {jwt}", so we need to use a regex